### DISCUSSION TITLE
全台首間校園虛擬電廠上線 靠AI降低30%用電量 還能供綠電

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#703] [General]
https://e-info.org.tw/node/230876
為了自己的環境找享而不是視而不見，從現在做起

**Comment by Valentinaa0214** [comment]
從節省能源開始著手，降低高功率電器使用，並轉換太陽能板及蓄電池的供電真的是很聰明的作法。
希望這套AI能源管理系統能在未來普及於各處，而不僅是校園，解決台灣目前電量不足的問題。

**Comment by yjpxi** [comment]
我的高中在我就讀的時候，我們學校每個學期都會超標，要多罰錢，總務處都會發通知叫每班省電，但是卻沒有很大的效果，所以如果虛擬電廠在我們高中使用的話我覺得會有很大的幫助。
